developer/openshift/dev_setup.sh

```diff
@@ -275,6 +275,9 @@ install_pipeline_service() {
     --kustomization "git::$GIT_URL/developer/openshift/gitops/argocd?ref=$GIT_REF" |
     indent 2
 
+  echo "- Installing local postgres DB for tekton results:"
```

Comment:
Why isn't that a kustomization applied to pipeline-service? or minio?

Reply:
I thought about this not as "kustomizing" an existing component (database) but as installing a new one as a dependency of pipeline-service, so the idea is to be explicit about its installation.
Maybe it is more readable to have it as kustomization of "pipeline-service" ? I can work on it

Reply:
After having a closer look, trying a kustomize with pipeline-service will be messed by
https://github.com/openshift-pipelines/pipeline-service/blob/fe7507dafaa8f77bc5d1fafe2764df1c66a43d07/developer/openshift/dev_setup.sh#L286
Maybe we need to have both (the db and current-branch related modifs )in a kustomize in a subsequent PR

Reply:
Fine, we can clean it up later if needed.